hypervisor/src/arch/x86/emulator/mod.rs

```diff
@@ -12,6 +12,7 @@ use crate::arch::x86::{
     StandardRegisters,
 };
 use anyhow::Context;
+pub use iced_x86::Register;
```

Comment:
Would you be able to get away with `use iced_x86::Register` in mshv/mod.rs instead?

Reply:
Yes. It works.